Objects/listobject.c

```diff
@@ -1315,9 +1315,15 @@ list_extend_set(PyListObject *self, PySetObject *other)
 {
     Py_ssize_t m = Py_SIZE(self);
     Py_ssize_t n = PySet_GET_SIZE(other);
-    if (list_resize(self, m + n) < 0) {
+    Py_ssize_t r = m + n;
```

Comment:
Well..... strictly speaking we might have an overflow here (but we already had the possibility of having one) but let's decide whether it's worth checking or not the bounds elsewhere.